waku/waku_rendezvous/protocol.nim

```diff
@@ -0,0 +1,214 @@
+{.push raises: [].}
```

Comment:
> It's true but that it's not protocol per say but neither is Relay.
> 
> I don't know how to inherit `Rendezvous` otherwise I would. This is a workaround.

Yes, you are absolutely right. Kindly disregard my previous comment :)